packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -2654,7 +2674,7 @@ const schema: SchemaType<DbUser> = {
   allowDatadogSessionReplay: {
     type: Boolean,
     optional: true,
-    nullable: true,
+    nullable: true, //TODO not-null: I assume intentional
```

Comment:
This field should be checked by the EA forum

Reply:
(_and_ it gets clobbered by spreading `...schemaDefaultValue(...)` afterwards, so if it should be maintained, it should be moved down to be after that spread, so it clobbers it instead of the other way around)